h2o-core/src/main/java/water/rapids/Env.java

```diff
@@ -297,6 +298,9 @@ static void init(AstPrimitive ast, String name) {
     init(new AstRepLen());
     init(new AstSeq());
     init(new AstSeqLen());
+    
+    // Statistics
+    init(new AstPearson());
```

Comment:
This already exist in AstCor

Reply:
Fail. I could've looked at it a little bit more. Would we still consider using parts of my code ? Some time in future? Or should I close it ?

- It's API is design to work on two columns only for simplicity, but it can easily be adapted for `n` columns, just like the original `AstCorrelation`.
- It doesn't need to duplicate any data to fight NaNs present in the dataset - the `array` method of `AstCorrelation` does that for NAs. It calculates means without NaN rows in a distributed manner.
- My code could be optimized to use `Vec.mean` when there are no NAs in the vec, which is an information give "for free" from vec's stats.

Reply:
Good point Michalk.  It is easy to miss since there is no Pearson in the title of the rapids call.

Reply:
I agree with Pavel with extending it to do multiple correlations instead of two columns can be useful.